src/bigstring_unix.mli

```diff
@@ -6,6 +6,15 @@ type t = (char, Bigarray.int8_unsigned_elt, Bigarray.c_layout) Bigarray.Array1.t
     on [Bigstring].
     @since 0.3 *)
 
+(** {2 I/O} *)
+
+(** These I/O functions are missing from the Bigarray library.
```

Comment:
not missing anymore, are they? :)

Reply:
They will still be missing from the ``Bigarray``, even when we add the to the ``Bigstring`` library. Should I remove this sentence?